### ISSUE TITLE
[BUG] Misleading messages in `crc setup`

### ISSUE THREAD

**Issue by cfergeau** [crc-org/crc#4079]
### General information

  * OS: Windows
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it: Yes
  * Running CRC on: VM

## CRC version
```bash
2.33
```
  
This is a newly installed Windows 11 version. Running `crc setup` told me my user was not part of the `crc-users` and `Hyper-V Admin` groups which is expected. A "fix" function was run as I got an UAC prompt, and then crc told me to reboot. I rebooted, re-ran `crc setup`, and I got exactly the same failure and the same instructions.

After discussing this with the team, I realized on Windows `crc setup` is not enough to get the machine fully configured for crc use. crc expects that it was installed using the Windows installer (or chocolatey). After installing with the Windows installer instead of `make && crc setup`, `crc setup` ran successfully.

If the `crc-users` group is expected to be created by the installer, then https://github.com/crc-org/crc/blob/fe2a917667675903da45838e236136c58c7e4f5c/pkg/crc/preflight/preflight_checks_windows.go#L160-L168 should be removed from the preflights, and idealy we'd also adjust the error message to say that crc was incorrectly installed, and that it's expected to be installed through the MSI.

**Comment by cfergeau** [comment]
`fixUserPartOfCrcUsersAndHypervAdminsGroup() ` is needed for chocolatey
However, there's a preflight check for the `crc-users` group, but it's only run for `crc start`, not  for `crc setup`. I think it should run in both cases, which would have given a nicer failure.
https://github.com/crc-org/crc/blob/b9337534a27f870f51b7dfc6292481ba39f5a2ab/pkg/crc/preflight/preflight_windows.go#L159

**Comment by gbraad** [comment]
@redbeam another goodfirstissue...

Note: this needs you to run Windows, ... so, good point to start a discussion.

**Comment by cfergeau** [comment]
> @redbeam another goodfirstissue...
> 
> Note: this needs you to run Windows, ... so, good point to start a discussion.

Ah hrm, I took care of this already in https://github.com/crc-org/crc/pull/4082 :(

**Comment by gbraad** [comment]
I noticed; in future, please consider 'trivial' and 'good first issues' that have a low(er) priority to be available for others to work on.

**Comment by cfergeau** [comment]
Fwiw, I don't think it was that trivial, but could have definitely been a good first issue with some guidance.
